test/integration/singlecluster/controller/core/localqueue_controller_test.go

```diff
@@ -164,8 +164,8 @@ var _ = ginkgo.Describe("Queue controller", ginkgo.Ordered, ginkgo.ContinueOnFai
 					{
 						Type:    kueue.LocalQueueActive,
 						Status:  metav1.ConditionTrue,
-						Reason:  "Ready",
-						Message: "Can submit new workloads to clusterQueue",
+						Reason:  kueue.LocalQueueActive,
```

Comment:
please use string literals in the test - whatever kueue.LocalQueueActive evaluates to

Reply:
Done